code/SPUC/spuc.py

```diff
@@ -34,7 +34,7 @@
 
 # ------------------------------------------------------------------------------
 # Endpoint for exporting the unicorn sightings if the EXPORT environment variable is set to True
-if os.environ.get("EXPORT").lower() == "true":
+if os.environ.get("EXPORT") == "True":
```

Comment:
Is this more restricting intentionally, or was it not working?

Reply:
Not working :'(